feat: MinK GA release (#5704)
Kong/docs.konghq.com: app/konnect/mesh-manager/index.md

---
title: About Mesh Manager
content_type: explanation

> beta: true

Reviewer: ```suggestion
beta: false
```

* **Kong-managed global control plane:** By creating your service mesh in {{site.konnect_short_name}}, your global control plane is managed by Kong. 
* **All entities in one place:** You can view all your information, such as entities from Kong Ingress Controller (KIC) for Kubernetes, {{site.konnect_short_name}} managed entities, and now service mesh data all from one central platform. 
* **Managed UI wizard setup:** {{site.konnect_short_name}} simplifies the process of creating a service mesh by providing a setup wizard in the UI that guides you through the configuration steps. 

### View service mesh entities

After your service mesh is deployed in {{site.mesh_product_name}}, the following information will be displayed in Mesh Manager for each control plane:

* Meshes and data plane proxies with [mTLS](/mesh/latest/policies/mutual-tls/)
* RBAC
* Zone control planes
* [Zone Ingresses](/mesh/latest/explore/zone-ingress/)
* [Zone Egresses](/mesh/latest/explore/zoneegress/)
* Services associated with your service mesh
* [Gateways](/mesh/latest/explore/gateway/) associated with your service mesh
* Policies

> ![control plane dashboard](/assets/images/docs/konnect/konnect-control-plane-dashboard.png)

Reviewer 2: Can we stay away from screenshots at least until the redesign is done (otherwise they are going to be outdated as soon as they land)

---
title: About Mesh Manager
content_type: explanation
---

> [Mesh Manager](https://cloud.konghq.com/mesh-manager) in {{site.konnect_product_name}} allows you to create, manage and view your {{site.mesh_product_name}} [service meshes](/mesh/latest/introduction/what-is-a-service-mesh/) in the {{site.konnect_short_name}} UI and using the {{site.konnect_short_name}} platform.

Reviewer: ```suggestion
[Mesh Manager](https://cloud.konghq.com/mesh-manager) in {{site.konnect_product_name}} allows you to create, manage and view your {{site.mesh_product_name}} [service meshes](/mesh/latest/introduction/about-service-meshes/) in the {{site.konnect_short_name}} UI and using the {{site.konnect_short_name}} platform.
```

> [Mesh Manager](https://cloud.konghq.com/mesh-manager) in {{site.konnect_product_name}} allows you to create, manage and view your {{site.mesh_product_name}} [service meshes](/mesh/latest/introduction/what-is-a-service-mesh/) in the {{site.konnect_short_name}} UI and using the {{site.konnect_short_name}} platform.

Reviewer 3: ```suggestion
[Mesh Manager](https://cloud.konghq.com/mesh-manager) in {{site.konnect_product_name}} allows you to create, manage, and view your {{site.mesh_product_name}} [service meshes](/mesh/latest/introduction/what-is-a-service-mesh/) in the {{site.konnect_short_name}} UI and using the {{site.konnect_short_name}} platform.
```

## Supported installation options

{{site.konnect_short_name}} supports the following installation options for {{site.mesh_product_name}} zones:

* Kubernetes
* Helm
* OpenShift
* Docker
* Amazon Linux
* Red Hat
* CentOS
* Debian
* Ubuntu
* macOS

## View service mesh entities

After your mesh is deployed in {{site.mesh_product_name}}, the following information will be displayed in Mesh Manager for each control plane:

* Meshes and data plane proxies with [mTLS](/mesh/latest/policies/mutual-tls/)
* RBAC
* Zone control planes
* [Zone Ingresses](/mesh/latest/explore/zone-ingress/)
* [Zone Egresses](/mesh/latest/explore/zoneegress/)
* Services associated with your mesh
* [Gateways](/mesh/latest/explore/gateway/) associated with your mesh
* Policies

![mesh control plane dashboard](/assets/images/docs/konnect/konnect-mesh-control-plane-dashboard.png)
> _**Figure 1:** Example control plane dashboard with several zones and services, a service mesh, and data plane proxies._

## Mesh Manager RBAC

> Mesh Manager has it's own role-based access control (RBAC) settings that are separate from the [{{site.konnect_short_name}} RBAC settings](/konnect/org-management/teams-and-roles/roles-reference/). The Mesh Manager RBAC settings are specific to the meshes and mesh policies in Mesh Manager. 

Reviewer: ```suggestion
Mesh Manager has its own role-based access control (RBAC) settings that are separate from the [{{site.konnect_short_name}} RBAC settings](/konnect/org-management/teams-and-roles/roles-reference/). The Mesh Manager RBAC settings are specific to the meshes and mesh policies in Mesh Manager. 
```
